tests/tests.src.js

```diff
@@ -6620,6 +6620,70 @@
 			Error,
 			'Parse error: forbidden code point when `allowUnsafeSymbols: true` and `strict: true`'
 		);
+		equal(
+			he.encode('\xE4\xF6\xFC\xC4\xD6\xDC', { 'decimal': true}),
+			'&#228;&#246;&#252;&#196;&#214;&#220;',
+			'encode to decimal HTML entities'
+		);
+		equal(
+			he.encode('\xE4\xF6\xFC\xC4\xD6\xDC', { 'decimal': true, 'useNamedReferences': true }),
+			'&auml;&ouml;&uuml;&Auml;&Ouml;&Uuml;',
+			'encode to named HTML entities whereby `useNamedReferences` takes precedence over `decimal`'
+		);
+		equal(
+			he.encode('a<b', { 'decimal': true, 'encodeEverything': true }),
+			'&#97;&#60;&#98;',
+			'`encodeEverything` to decimal HTML entities'
+		);
+		equal(
+			he.encode('\0\x89', {'decimal': true}),
+			'\0\x89',
+			'Does not encode invalid code points whose character references would refer to another code point, neither if `decimal`: true is used'
+		);
+		equal(
+			he.encode('\0\x89', { 'decimal': true, 'encodeEverything': true }),
+			'\0\x89',
+			'Does not encode invalid code points whose character references would refer to another code point, even when `encodeEverything: true` and `decimal: true` is used'
+		);
+		equal(
+			he.encode('foo\xA9<bar\uD834\uDF06>baz\u2603"qux', { 'allowUnsafeSymbols': true }),
+			'foo&#169;<bar&#119558;>baz&#9731;"qux',
```

Comment:
This test fails on my local machine. I could not find the reason, since grunt does not print all test output (terminates with: ```Warning: stdout maxBuffer exceeded. Used --force, continuing.```). 

1) Do others have an idea why this test fails? I only replaced the hexadecimal by the decimal representation.
2) How to get all test outputs? Workaround for ```the stdout maxBuffer exceeded``` known? I tried already ```--force``` but without luck.